doc/en/getting-started.rst

```diff
@@ -185,6 +185,21 @@ You can find out what kind of builtin :ref:`fixtures` exist by typing::
 
     pytest --fixtures   # shows builtin and custom fixtures
 
+Contributing tests to an existing project
```

Comment:
The `pip install -e`/`setup.py develop` concept is mentioned there already, and I don't think the whole "contributing to an existing repo" concept fits there.   Hmm.

The only other place the develop concept is mentioned is the [nose page](http://doc.pytest.org/en/latest/nose.html).